src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/RazorIntegrationTestBase.cs

```diff
@@ -218,7 +218,7 @@ protected CompileToCSharpResult CompileToCSharp(string cshtmlRelativePath, strin
 
                 // Add the 'temp' compilation as a metadata reference 
                 var references = BaseCompilation.References.Concat(new[] { tempAssembly.Compilation.ToMetadataReference() }).ToArray();
-                projectEngine = CreateProjectEngine(RazorConfiguration.Default, references);
+                projectEngine = CreateProjectEngine(Configuration, references);
```

Comment:
The two changes in this file from `RazorConfiguration.Default` to `Configuration` are to make the test behavior consistent. If a test subclass overrides the pre-existing `Configuration` property (which defaults to `RazorConfiguration.Default`) then we really should use that override instead of certain bits of code being hardcoded to use `RazorConfiguration.Default` directly.

This change is needed for the test that demonstrates the output on 3.x.